doc/admin/predeployment_check.md

```diff
@@ -71,7 +71,7 @@ to be used by DAOS. This can be done using the following command (<ifaces> must
 be replaced with the interface names):
```

Comment:
this probably best moved to line 96, as there are no longer mentions of <ifaces> before that

Reply:
Agreed. @cdurf1 please fix that in another PR.